plugins/hosts/linux/cap/nfs.rb

```diff
@@ -261,6 +261,28 @@ def self.nfs_running?(check_command)
           Vagrant::Util::Subprocess.execute(*Shellwords.split(check_command)).exit_code == 0
         end
 
+        def self.modinfo_path
+          if !defined?(@_modinfo_path)
+            @_modinfo_path = Vagrant::Util::Which.which("modinfo")
+
+            if @_modinfo_path.to_s.empty?
+              folders = ["/sbin"]
+              folders.each do |folder|
```

Comment:
I'm re-reading this now...are we expecting more folders here to be searched besides `/sbin`? :thinking: if not, we could remove the block enumerator and just check the path against `/sbin`.

Reply:
Good point!  I patterned this after plugins/hosts/gentoo/cap/nfs.rb, but in this case we're really only checking one place.

This could be made even simpler if we say "use `/sbin/modinfo` if it exists, otherwise use `modinfo`" -- what do you think?

Reply:
Aha, I see :thinking: Yeah I think in this case since we know we explicitly want to just check `/sbin` we can remove the iterator and just check that here. If we need to add more paths in the future we can change it then. I'd say just replace that, but otherwise leave the method as is so that we can first check if `Which` gives us a result, if not check`/sbin` and as a last resort just try `modinfo`